cves/kernel/CVE-2014-1739.yml

```diff
@@ -235,8 +234,8 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
+  - commit: c06ca8f92b0b23dfd209fea27d71046c9c776910
+    note: The commit addresses the fact that a buffer does not get cleared in the same method as the vulnerability which would result in a memory leak as well but the vulnerability at hand does not get addressed.
```

Comment:
Might not be within the scope of the project, but it'd be interesting to know how the other vulnerability was found.  What was done that couldn't find the other?